src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -85,6 +85,11 @@
     private static final String NOT_EXISTING_FILE = "target/not_existing.xml";
     private static final String FAILURE_PROPERTY_VALUE = "myValue";
 
+    @Override
+    protected String getPackageLocation() throws IOException {
+        return "com/puppycrawl/tools/checkstyle";
```

Comment:
@subkrish I thought `ant` package was re-organized because some are in special folder. This isn't the case.
You can ignore this comment as it will be handled as part of #3891 .